OPERATOR.rst

```diff
@@ -156,28 +156,40 @@ label from blocking PR. Last, remove the blocking relationship.
 Upgrading GitLab
 ^^^^^^^^^^^^^^^^
 
-Occasionally it falls on the operator to upgrade the Azul GitLab instance. If
-the current major version is ``n`` and the latest available major version is
-greater than ``n+1`` (i.e. upgrading directly to the latest version would skip
-one or more major versions) then multiple successive upgrades must be made, such
-that no upgrade skips a major version. For example, if the current version is
-13.x.y and the latest available version is 15.x.y, then one would first upgrade
-to 14.x.y and then repeat the process to upgrade to 15.x.y.
-
-Before any changes are applied, stop the instance (do not terminate) and create
-a snapshot of its EBS volume. Edit ``terraform/gitlab/gitlab.tf.json.template.py``,
-updating the versions of the docker images for ``gitlab-ce`` and
-``gitlab-runner``. Then run::
+Operators must check for updates to the `GitLab Docker image`_ and the
```

Comment:
A quicker way to check for this https://gitlab.dev.singlecell.gi.ucsc.edu/help

Reply:
Wanted to reference the docker image release page that way it's explicit where to find the new image/what the new image is.

Reply:
Nevertheless, please incorporate this in as a way of short-circuiting. There is no point in comparing the docker page against the TF source if the help page indicates that Gitlab is up-to-date.

Reply:
So to confirm, you want the operators to check that the GitLab Instance can be updated using the `/help` endpoint, and inspect if there are updates to the GitLab Runners by comparing the TF file to the docker release page?

Reply:
I want them to check daily (since it's so easy) for updates using /help and if that indicates that updates are available, install the latest version of Gitlab and the latest compatible version of the runner.